app/scripts/controllers/reviewRegistration.js

```diff
@@ -280,4 +280,158 @@ angular.module('confRegistrationWebApp')
+            else {
+              $scope.spouseSectionVisible = false;
+            }
+          }
+        }).catch(function(response){
+        console.log('Failed to get all registered users for current conference. Status = ' + response.status);
+      });
+    };
+
+    //Click event for 'Include Spouse' button
+    $scope.includeSpouseInRegistration = function () {
+
+      //Generate new local UUID used for registrantId
+      var _newRegistrantId = uuid();
+
+      //Payload for new spouse registration
+      var _newSpouseRegistration =
+        {
+          id: $scope.spouseRegistration.id,
+          conferenceId: $scope.currentRegistration.conferenceId,
+          registrants: [{
+            id: _newRegistrantId,
+            registrationId: $scope.spouseRegistration.id,
+            registrantTypeId: $scope.currentRegistration.registrants[0].registrantTypeId,
+            answers: []
+          }]
+        };
+
+      var _answers = '';
+
+      //Get all answers
+      _.forEach($scope.currentRegistration.registrants[0].answers, function(value) {
+        if (typeof(value.value) === 'object') {
+          value.value = null;
+        }
+        _answers = _answers +
+          '{' +
+            'id: ' + uuid() + ',' +
+            'registrantId: ' + _newRegistrantId + ',' +
+            'blockId: ' + value.blockId + ',' +
+            'value: ' + value.value + ',' +
+            'amount: ' + value.amount +
+            // any other fields from the answer must be copied as well
+          '}' + ',';
+      });
+
+      //Payload to add registrations to new spouse registration
+      var _newRegistrants =
+      {
+        id: _newRegistrantId,
+        registrationId: $scope.spouseRegistration.id,
+        registrantTypeId: $scope.currentRegistration.registrants[0].registrantTypeId,
+        answers: [_answers],
+        firstName: $scope.currentRegistration.registrants[0].firstName,
+        lastName: $scope.currentRegistration.registrants[0].lastName,
+        email: $scope.currentRegistration.registrants[0].email
+      };
+
+      //Add new registration
+      //@Path("/registrations/{registrationId}")
+      $http.put('/registrations/' + $scope.spouseRegistration.id, _newSpouseRegistration)
+        .then(function () {
+          //success
+        }).catch(function () {
+        alert('An error occurred while adding new spouse registration.');
+      });
+
+      //Add registrants to new registration
+      //@Path("/registrants/{registrantId}")
+      $http.put('/registrants/' + _newRegistrantId, _newRegistrants)
+        .then(function () {
+          //success
+        }).catch(function () {
+        alert('An error occurred while adding registrants to new spouse registration.');
+      });
+
+      //Delete existing registration
+      //@Path("/registrants/{registrantId}")
+      $http.delete('/registrants/' + $scope.spouseRegistration.id, $scope.spouseRegistration)
```

Comment:
Same as above; this request should be made in the `.then` callback of the preceding request.